src/css/doc.css

```diff
@@ -494,15 +494,16 @@ ul ul ul {
 }
 
 .doc .quoteblock {
-  margin-top: var(--base-medium-space);
+  margin-top: var(--base-extra-large-space);
 }
 
 .doc .abstract blockquote {
-  font-size: 0.9375rem;
+  /* font-size: 0.9375rem; */
   margin: 0;
-  font-weight: var(--weight-light);
-  border-left: 2px solid var(--color-brand-gray5);
+  /* font-weight: var(--weight-light);
+  border-left: 2px solid var(--color-brand-gray5); */
   padding-left: 1.125rem;
```

Comment:
Looking again at the mockup, I wonder if we could remove the padding, and have this aligned like a normal paragraph?

Reply:
+1